packages/nimble-components/src/table-column/enum-text/tests/table-column-enum-text.spec.ts

```diff
@@ -363,6 +366,49 @@ describe('TableColumnEnumText', () => {
             }
         });
 
+        describe('invalid mappings', () => {
```

Comment:
I don't think this `describe` block is helpful from an organizational perspective. The test would read "validation invalid mappings is invalid with icon or spinner mappings", as opposed to "validation is invalid with icon or spinner mappings". Seems like we can just discard this inner `describe` block, and just move all of the helper functions/types as you see fit (which may be nowhere).